sdk/examples/how_tos/client/get_outputs.rs

```diff
@@ -28,12 +28,7 @@ async fn main() -> Result<()> {
         .await?;
 
     // Take the address from command line argument or use a default one.
-    let address = Bech32Address::try_from_str(
-        std::env::args()
```

Comment:
Not sure I would have removed the arg tbh

Reply:
Yeah thought about it too. Wasn't sure if there would be any obligations from your side.

Reply:
So will remove it

Reply:
Add it back you mean?

Reply:
Reverted